cypress/component/ERACommons/era_commons.spec.tsx

```diff
@@ -78,27 +84,24 @@ describe('ERA Commons Component', function () {
   })
 
   it('renders a populated ERA Commons component after having authenticated with NIH', function () {
-    const clonedResearcher = JSON.parse(JSON.stringify(researcher))
+    const clonedResearcher: Researcher = { ...researcher }
     clonedResearcher.eraCommonsId = 'testing'
     const iat = new Date().getTime()
-    const exp = iat + (30 * 24 * 60 * 60 * 1000) // iat + 30 days
+    const exp = iat + (30 * 24 * 60 * 60 * 1000)
```

Comment:
Retain comment from maintainability:

```suggestion
    const exp = iat + (30 * 24 * 60 * 60 * 1000)  // iat + 30 days
```